ex36.py

```diff
@@ -0,0 +1,39 @@
+#while True:
+#	print "priti"
+
+def printStr(str):
+	print str
+	return
+
+def change( mylist ):
+	mylist.append([1,2,3,4])
+	print "Values inside the function: ", mylist
+
+def changeme( mylist ):
+	mylist = [1,2,3,4]
+	print "Values inside the function: ", mylist
+
+def printinfo( name, age ):
+	print "Name: ", name
+	print "Age ", age
+
+def printValue(name, age = 34):
+	print "Name: " , name
+	print "Age: " , age
+
+def addNumbers(*vars):
+	for i in vars:
+		print "Number is : %d" % i
+
+addNumbers(34,56)
+addNumbers(55, 56, 57, 58, 59, 60)
+print printStr("Calling printstr function")
+mylist = [10,20,30]
+changeme(mylist)
+print "Values outside the function: ", mylist
+printStr("This print an string")
+change([2, 33, 55])
+printinfo( age=50, name="miki" )
+printValue(age = 30, name = "Priti")
```

Comment:
As per PEP8 avoid spaces between kwargs (key word arguments) in python. eg: (age=30) and not (age = 30). Also, avoid spaces for method parameters while defining them. eg: 'def printinfo( name, age ):' should be 'printinfo(name, age):'